lib/screens/explore_screens/explore/interactor/viewmodels/explore_bloc.dart

```diff
@@ -10,6 +10,7 @@ class ExploreBloc extends Bloc<ExploreEvent, ExploreState> {
   ExploreBloc() : super(ExploreState.initial()) {
     on<OnExploreCardTapped>((event, emit) => emit(state.copyWith(pageCommand: NavigateToRoute(event.route))));
     on<OnBuySeedsCardTap>((event, emit) => emit(state.copyWith(pageCommand: NavigateToBuySeeds())));
+    on<OnFlagUserTap>((event, emit) => emit(state.copyWith(pageCommand: ShowUserFlagInformation())));
```

Comment:
You don't use the event parameter
`(_, emit)`
Same for the intruction above, my mistake could you remove both plz.

Reply:
done, This should also be caught using lint.